File: MODUL-4/SOAL 1/PRAK401-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

void main() {
    int kelipatan;
    char simbol;
    printf("Masukkan bilangan kelipatan dan simbol : ");
    scanf("%d %c", &kelipatan, &simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0)
            printf("%c ", simbol);
        else
            printf("%d ", i);
    }
}

File: MODUL-4/SOAL 1/PRAK401-2410817120017-RIKAFAULIANARAHMI.py
kelipatan, simbol = input("Masukkan bilangan kelipatan dan simbol : ").split()
kelipatan = int(kelipatan)

for i in range(1, 51):
    if i % kelipatan == 0:
        print(simbol, end=" ")
    else:
        print(i, end=" ")
